src/components/FooterWrapper.js

```diff
@@ -31,8 +31,12 @@ const FooterWrapper = ({ children }) => {
         footerLogo()
       ]),
       a({ href: Nav.getLink('privacy'), style: styles.item }, 'Privacy Policy'),
+      div({ style: styles.item }, '|'),
       a({ href: Nav.getLink('terms-of-service'), style: styles.item }, 'Terms of Service'),
       div({ style: styles.item }, '|'),
+      a({ href: 'https://support.terra.bio/hc/en-us/articles/360030793091-Terra-FireCloud-Security-Posture', target: '_blank', style: styles.item },
```

Comment:
We should use `Utils.newTabLinkProps` here instead of applying `target: '_blank'` directly. (See other usages for examples)